docs/tools/index.txt

```diff
@@ -5,69 +5,47 @@ Tools
 There are several tools that play well with CrateDB, from development utilities
 to data visualization tools. We even use some of them ourselves!
 
-This section of the documentation covers some of the tools available. If you
-know of any other tools that work well with CrateDB, please `get in touch
-<mailto:office@crate.io>`_.
-
-.. SEEALSO::
-
-    The `client tools`_ category on our blog.
-
-.. toctree::
-   :hidden:
-
-   streamsets
+- **Pentaho**
 
-Data Visualization
-==================
+  `Pentaho`_ prepares and blends data, delivering business analytics from any
+  source. You can connect to CrateDB clusters by using `Petaho Kettle`_ and the
+  standalone version of `our JDBC driver`.
 
-Pentaho
--------
+- **StreamSets Data Collector**
 
-Pentaho prepares and blends data, delivering business analytics from any
-source. You can connect to CrateDB clusters by using the standalone version of
-`our JDBC driver <https://crate.io/docs/reference/jdbc/>`_.
+  The `StreamSets`_ Data Collector is a lightweight, powerful engine that
+  streams data in real time. Read `the manual`_ on how to use CrateDB as an
```

Comment:
`Read the manual on how to use CrateDB as an origin` - does this suggest that there is a manual for using CrateDB as an origin and destination? 

Maybe `read the StreamSets manual where CrateDB can be set as an origin..`?

Reply:
another good catch! I got the link wrong. pushed up a fixup that corrects the link text and the link URL